src/App.tsx

```diff
@@ -14,11 +14,148 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface State {
+  goods: string[],
+  isStarted: boolean,
+  isReversed: boolean,
+  sortBy: string,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: goodsFromServer,
+    isStarted: false,
+    isReversed: false,
+    sortBy: '',
+  };
+
+  started = () => {
+    this.setState((prevState) => ({
+      isStarted: !prevState.isStarted,
+    }));
+  };
+
+  reverseGoods = () => {
+    this.setState(prevState => ({
+      isReversed: !prevState.isReversed,
+    }));
+  };
```

Comment:
This state field is redundant.
On pressing this button you just need to perform some actions on the goods stored in state.